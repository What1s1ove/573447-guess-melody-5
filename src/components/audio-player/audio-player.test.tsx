import * as React from 'react';
import { render } from 'enzyme';
import AudioPlayer from './audio-player';

describe(`AudioPlayer component`, () => {
  it(`should render AudioPlayer component`, () => {
    const component = render(
      <AudioPlayer
        isPlaying={false}
        isLoading={false}
        onPlayBtnClick={jest.fn()}
        src="play.src"
      />
    );

    expect(component).toMatchSnapshot();
  });
});
