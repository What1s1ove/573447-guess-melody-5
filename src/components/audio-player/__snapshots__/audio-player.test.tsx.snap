// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AudioPlayer component should render AudioPlayer component 1`] = `
Array [
  <button
    aria-label="Play track"
    class="track__button track__button--play"
    type="button"
  />,
  <div
    class="track__status"
  />,
]
`;
